modules/xmpp/JingleSessionPC.js

```diff
@@ -57,6 +58,92 @@ function getEndpointId(jidOrEndpointId) {
     return Strophe.getResourceFromJid(jidOrEndpointId) || jidOrEndpointId;
 }
 
+/**
+ * Add "source" element as a child of "description" element.
+ */
```

Comment:
Please add a one line desc about the parameters.

Reply:
done.